lib/game.rb

```diff
@@ -0,0 +1,112 @@
+require 'pry'
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+  WIN_COMBINATIONS = [
+  [0,1,2], # top row
+  [3,4,5], # middle row
+  [6,7,8], # bottom row
+  [0,3,6], # left column
+  [1,4,7], # mid column
+  [2,5,8], # right column
+  [0,4,8], # diag 1
+  [6,4,2] # diag 2
+]
+
+  def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    if @board.turn_count.odd?
+      @player_2
+    else
+      @player_1
+    end
+  end
+
+  def draw?
+    if full? && !won?
+      true
+    else
+      false
+    end
+  end
+
+  def full?
+    @board.cells.all? do |spot|
+      spot != " "
+    end
+  end
```

Comment:
You shouldn't need to redefine this method...is it possible to use `board.full?`?